crates/cxx-qt-lib/src/types/mod.rs

```diff
@@ -6,7 +6,9 @@
 mod qbytearray;
 pub use qbytearray::QByteArray;
 
+#[cfg(feature = "qt_gui")]
 mod qcolor;
```

Comment:
could also change the folder / module structure to be
```
cxx-qt-lib/
  - core/
    - ...
  - gui/
    - ...
```

And then have `cxx_qt_lib::core::QPoint` and `cxx_qt_lib::gui::QColor` ?

Reply:
Done